### ISSUE TITLE
你好! 有问题请教你

### ISSUE THREAD

**Issue by jideee** [johsnows/ssd-migo-mobilentv3#1]
请问 你这个跟经过预训练了吗 ?
我是从另一个人哪里 看到你评论  "V3是否有在imagenet预训练过？没有预训练的效果很差"  而过来的..

我也是用他的程序跑出来mAP很低 ,,想找个预训练模型.可惜找不到匹配的.........
请问你这个程序加入了预训练了吗?  你这个跑出来的mAP是多少呢?

还有的就是:
这是你给出的
train:
python train_ssd.py --datasets /userhome/VOCdevkit/VOC2007 /userhome/VOCdevkit/VOC2012 --validation_dataset /userhome/VOCdevkit/VOCtest2007  --net migo --scheduler cosine --lr 0.01 --t_max 200 --validation_epochs 5 --num_epochs 200 --structure_path structures/dynamic_sng_V3_600/dynamic_SNG_V3_ofa_imagenet_width_multi_1.3_600_B.json --migo_size 2

从这儿开始:   --structure_path structures/dynamic_sng_V3_600/dynamic_SNG_V3_ofa_imagenet_width_multi_1.3_600_B.json --migo_size 2
能告知下是什么意思吗?  谢谢

望回答~~ 非常感谢!

**Comment by johsnows** [comment]
目前v3没有改成能加预训练的，但是后面我可以改，目前我主要在做coco上跑的，coco讲道理大家都是不预训练的。你可以忽略那个structure_path的部分，目前还在做  只关注v3如何使用就行。之后会加上在imagenet上预训练的v3在voc上train的模型，以及v2,v3在coco上无预训练train。













在 2020-07-19 15:12:23，"jideee" <notifications@github.com> 写道：

   请问 你这个跟经过预训练了吗 ?


我是从另一个人哪里 看到你评论 "V3是否有在imagenet预训练过？没有预训练的效果很差" 而过来的..

我也是用他的程序跑出来mAP很低 ,,想找个预训练模型.可惜找不到匹配的.........
请问你这个程序加入了预训练了吗? 你这个跑出来的mAP是多少呢?

还有的就是:
这是你给出的
train:
python train_ssd.py --datasets /userhome/VOCdevkit/VOC2007 /userhome/VOCdevkit/VOC2012 --validation_dataset /userhome/VOCdevkit/VOCtest2007 --net migo --scheduler cosine --lr 0.01 --t_max 200 --validation_epochs 5 --num_epochs 200 --structure_path structures/dynamic_sng_V3_600/dynamic_SNG_V3_ofa_imagenet_width_multi_1.3_600_B.json --migo_size 2

从这儿开始: --structure_path structures/dynamic_sng_V3_600/dynamic_SNG_V3_ofa_imagenet_width_multi_1.3_600_B.json --migo_size 2
能告知下是什么意思吗? 谢谢

望回答~~ 非常感谢!
